VariationalBayes/test_exponential_families.py

```diff
@@ -103,6 +104,45 @@ def get_r_matrix(mat):
             ef.expected_ljk_prior(lkj_param, df, v),
             atol=moment_tolerance)
 
+    def test_logitnormal_moments(self):
+        # log normal parameters
+        lognorm_means = np.random.random(5)
+        lognorm_infos = np.random.random(5)**2 + 1
+        alpha = 2 # dp parameter
+
+        # draw samples
+        num_draws = 10**5
+        samples = np.random.multivariate_normal(lognorm_means,
```

Comment:
If you want univariate normals, please do not use ```np.random.multivariate_normal```, especially not with such a giant covariance matrix..  Use ```np.random.normal```.